src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue

```diff
@@ -26,22 +26,26 @@
                class="td content clickAble">
             <span v-for="(singleLine, newIndex) in singleObject[1]"
                   :key="newIndex"
-                  :class="determinePlaceAndVisiblity(index, newIndex)"
-                  @click="singleObject[0] !== 'content' ? searchFromAllValues(singleObject[0], singleLine) : null">
-              {{ singleLine }} <br>
+                  
+                  :class="determinePlaceAndVisiblity(index, newIndex)">
+              <router-link v-if="singleObject[0] !== 'content'" :to="{ path: createLinkToValueSearch(singleObject[0], singleLine) }"> {{ singleLine }} </router-link>
+              <br>
             </span>
-            <button v-if="singleObject[1].length > arrayShownLimit"
+            <button v-if="
+                      single-object-1-length> arrayShownLimit"
```

Comment:
What is _single-object-1-length_ a reference to? :)

Reply:
Fixed, that was weird!